app/controllers/publications/overview/shortlist.js

```diff
@@ -0,0 +1,127 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class PublicationsOverviewShortlistController extends Controller {
+  @service publicationService;
+  @service router;
+  @service store;
+
+  @tracked isOpenNewPublicationModal;
+  pieceForPublication;
+  agendaitemForPublication;
+  decisionActivityForPublication;
+  caseForPublication;
+  mandateesForPublication;
```

Comment:
should these be tracked? we are updating these constantly.
Or not needed if we don't use them in the template?

Reply:
Only used in JS so tracking not necessary as far as I understand.